mm2src/coins/test_coin.rs

```diff
@@ -80,7 +80,7 @@ impl MarketCoinOps for TestCoin {
         unimplemented!()
     }
 
-    fn tx_enum_from_bytes(&self, _bytes: &[u8]) -> Result<TransactionEnum, MmError<TxMarshalingErr>> {
+    fn tx_enum_from_bytes(&self, bytes: &[u8]) -> Result<TransactionEnum, MmError<TxMarshalingErr>> {
```

Comment:
Shouldn't this lead to a warning as the `bytes` argument isn't used?

Reply:
Yes, it's strange this doesn't happen :raised_eyebrow:. Will fix it anyways.